docs/source/examplecleanup.rst

```diff
@@ -6,55 +6,94 @@ End-to-End Example
 
 An End-to-End example walkthrough of a tartufo scan and the process of purging the dirty evil passwords that somehow ended up in your code commits. We will use an additional tool: ``BFG`` (https://rtyley.github.io/bfg-repo-cleaner/, more on this later!). 
 
-#. Use ``tartufo`` to scan your repository and find any secrets in its history!
 
-   Use what you've learned to scan your repo!
+#. Setup poetry if you want to use the most recent build
+
+   This project uses [Poetry] to manage its dependencies and do a lot of the heavy lifting. So you'll need to clone the tartufo repo and setup poetry!
+
+
+   .. code-block:: console
+
+      git clone git@github.com:godaddy/tartufo.git
+      # use a virtual environment!
+      python3.7 -m venv .venv
+      . .venv/bin/activate
+      
+
+   Next: [Install Poetry]  
+
+   Finally: Run ``poetry install``
+
+
```

Comment:
This does need to be effectively documented, but it doesn't really belong in this document; this should be moved to the Installation doc instead. The Contributing doc should also probably be updated to point to the Installation doc.

Also, you should not be explicitly creating/using a venv when using Poetry. Instead, you really should let Poetry manage the venv for you. 😄 

So like the Contributing doc in this branch currently shows, you just need to make sure Poetry itself is installed and then run `poetry install`.

Reply:
As the title indicates we're working with a non-complete build, leaving those references out seems likely to leaving things to change for the end user. If we need better install poetry info, the links provided can be augmented or changed to another page (once it exists)

Reply:
The thing is, this is only a temporary state until 2.0 is released, which is planned ideally for some time this month. This document should be written with the assumption that 2.0 has already been released, so that it doesn't have to be changed again once we release. Note that this document ONLY displayed in the "latest" version of the docs, not the "stable" version, which is what shows to users by default. If you go to https://tartufo.readthedocs.io/ right now, you will not see this document, because it has never been added to the `1.x` branch, which is where `stable` tracks, until 2.0 has been released.

Also, again, this document should only cover *usage*, with installation covered in the Installation doc. Otherwise we have duplicated documentation leading to potentially outdated documentation and even possible user confusion.